types/video.js/index.d.ts

```diff
@@ -6195,6 +6207,35 @@ export interface VideoJsPlayer extends videojs.Component {
 
     aspectRatio(): string;
 
+    /**
+     * Get the current audioOnlyMode state or set audioOnlyMode to true or false.
+     *
+     * Setting this to `true` will hide all player components except the control bar,
+     * as well as control bar components needed only for video.
+     *
+     * @param [value]
+     *         The value to set audioOnlyMode to.
+     *
+     * @return A Promise is returned when setting the state, and a boolean when getting
+     *         the present state
+     */
+    audioOnlyMode(value: boolean): void;
+
+    audioOnlyMode(): Promise<void> | boolean;
```

Comment:
```suggestion
    audioOnlyMode(): boolean;
```
https://github.com/videojs/video.js/pull/7647/files#diff-b62e95b3e474754b0b665ff01617843398a03815683899a9c79f141bab6a1fa7R2851